### DISCUSSION TITLE
Export CSS variables

### DISCUSSION THREAD

**Discussion by brunosarlo** [mikaelvesavuori/figmagic#168] [Q&A]
Hi!

I'm looking forward to have figmagic export css variables. I couldn't find an easy way to do this yet. 

It has already been proposed as feature: https://github.com/mikaelvesavuori/figmagic/projects/1#card-38295671

Any ideas?

Thanks!

**Comment by mikaelvesavuori** [comment]
Hey @brunosarlo!

This will sound really dumb but I never got informed something happened in the GitHub Discussions... Sorry about that!

Yeah, new feature development has absolutely slowed down for a bit. If I got some assistance I think I could look at resurrecting that one.

**Comment by mikaelvesavuori** [comment]
Added basic support in [version 4.5.9](https://github.com/mikaelvesavuori/figmagic/releases/tag/v4.5.9). See https://github.com/mikaelvesavuori/figmagic#switch-token-file-format
